src/locales/en.ts

```diff
@@ -29,6 +29,18 @@ const en = {
   errors: {
     somethingWrong: () => "Something went wrong...",
     imageUploadError: () => "Unable to upload image(s)",
+    posts: {
+      postCreationError: () => "Unable to create post", // I don't like this message but I can't think of another
+      postUpdateError: () => "Post could not be edited",
+    },
+    comments: {
+      commentCreationError: () => "Unable to post comment",
+      commentUpdateError: () => "Unable to update comment",
+    },
+    users: {
+      userCreationError: () => "There was an error creating your account",
+      userUpdateError: () => "There was an error updating your account",
+    },
```

Comment:
Can these be moved to their respective models, so `errors.posts` becomes `posts.errors` and so on?

Reply:
Please review 05b051c and let me know what you think!